es-design-system/pages/organisms/es-file-upload.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+    <div>
+        <h1>
+            File Upload
+        </h1>
+
+        <div class="mb-450">
+            <h2 class="mb-200">
+                File Upload
+            </h2>
+            <es-file-upload
+                :upload-urls="urls"
+                :max-file-size="20"
+                :file-types="['image/png', 'application/pdf']"
+                @fileSizeError="fileSizeError"
+                @fileTypeError="fileTypeError"
+                @readyToUpload="readyToUpload"
+                @uploadSuccess="uploadSuccess"
+                @uploadFailure="uploadFailure"
+                @fileDataRead="fileDataRead"
+                @uploadProgress="uploadProgress">
+                <template #cta>
+                    <h2 class="d-none d-md-inline-block mb-4 text-center">
+                        Drag and drop your files<br>or
+                    </h2>
+                    <p class="d-inline-block d-md-none">
+                        <b>Don't forget:</b> Make sure to upload an image of both sides of your bill.
+                    </p>
+                </template>
+                <template #helpText>
+                    <p class="d-none d-md-inline-block">
+                        Please upload your file as a PDF or JPG.
+                    </p>
+                    <p class="d-inline-block d-md-none mb-0">
+                        File types: PDF or JPG
+                    </p>
+                </template>
+            </es-file-upload>
+        </div>
+        <div class="mb-450">
+            <h2 class="mb-200">
+                File Upload Collapsed
+            </h2>
+            <p>
+                Not intended for mobile use
+            </p>
```

Comment:
This is next to impossible to enforce. Whats the rational? Outside of fixing the button wrapping it seems to work fine.
<img width="176" alt="image" src="https://github.com/EnergySage/es-ds/assets/1017057/bae57fd7-eb82-4f78-801d-b272e2b122d7">

Reply:
Agreed. Made this look acceptable on mobile!